lib/standards/index.js

```diff
@@ -0,0 +1,31 @@
+import ariaAttrs from './aria-attrs';
+import { clone } from '../core/utils';
+
+const origAriaAttrs = clone(ariaAttrs);
+const standards = {
+	ariaAttrs
+};
+
+// @see https://stackoverflow.com/a/59008477/2124254
+function merge(current, updates) {
+	for (const key of Object.keys(updates)) {
+		if (!current.hasOwnProperty(key) || typeof updates[key] !== 'object') {
+			current[key] = updates[key];
+		} else {
+			merge(current[key], updates[key]);
+		}
+	}
+	return current;
+}
+
+export function configureStandards(config) {
+	if (config.ariaAttrs) {
+		standards.ariaAttrs = merge(ariaAttrs, config.ariaAttrs);
```

Comment:
For safety, I think we should clone and freeze:

```suggestion
		const ariaAttrs = merge(ariaAttrs, clone(config.ariaAttrs))
		standards.ariaAttrs = deepFreeze(ariaAttrs);
```